lib/matplotlib/backend_bases.py

```diff
@@ -2791,6 +2791,12 @@ def mouse_move(self, event):
             except (ValueError, OverflowError):
                 pass
             else:
+                ax = event.inaxes
+                artists = ax.artists + ax.images + ax.lines
```

Comment:
Right, but the ordering is a stable sort so ties are broken by which was in the list first.  This is the source of at least one un-fixable (with out major overhaul of the internal data structure, see #3944)  bug ( #409 ).